Uno.Gallery/Views/SamplePages/TextBoxSamplePage.xaml

```diff
@@ -217,6 +217,45 @@
 
 							</StackPanel>
 						</smtx:XamlDisplay>
+
+						<smtx:XamlDisplay UniqueKey="TextBoxSamplePage_Material_Playground"
+										  smtx:XamlDisplayExtensions.Header="TextBox Playground"
+										  smtx:XamlDisplayExtensions.IgnorePath="XamlDisplay\StackPanel">
+
+							<StackPanel Spacing="10"
+										Padding="20"
+										HorizontalAlignment="Center"
+										Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+										Orientation="Vertical">
+								<TextBox Style="{StaticResource MaterialFilledTextBoxStyle}"
+										 x:Name="txtFilled"
+										 Width="270" />
+								<TextBox Style="{StaticResource MaterialOutlinedTextBoxStyle}"
+										 x:Name="txtOutlined"
+										 Width="270" />
+								<StackPanel HorizontalAlignment="Center"
+											Spacing="8"
+											Orientation="Horizontal">
+									<Button Content="Add Header"
+											Click="AddHeader" />
+									<Button Content="Add Placeholder"
+											Click="AddPlaceholder" />
+									<Button Content="Add Text"
+											Click="AddText" />
+								</StackPanel>
+
+								<StackPanel VerticalAlignment="Center"
+											Spacing="8"
+											Orientation="Horizontal">
+									<Button Content="Remove Header"
+											Click="RemoveHeader" />
+									<Button Content="Remove Placeholder"
+											Click="RemovePlaceholder" />
+									<Button Content="Remove Text"
+											Click="RemoveText" />
```

Comment:
sanity check: the event handler can actual bind from inside the data-template?

Reply:
Yes. We also use it on AnimationSamplePage: https://github.com/unoplatform/Uno.Gallery/blob/b3023aa0256e9f37d8aea24e9e02b312aad3e0b0/Uno.Gallery/Views/SamplePages/AnimationSamplePage.xaml#L127-L129